card/mod/05_email/lib/card/follow_option.rb

```diff
@@ -48,6 +50,15 @@ def follow_opts args
         add_option args, :main
       end
       
+      def follow_test opts={}, &block
+        codename = get_codename opts
+        Card::FollowOption.test[codename] = block
+      end
+      
+      def follow_test_option option, &block
+        Card::FollowOption.test_option[option] = block
+      end
+      
```

Comment:
could stand for some naming improvements.  the follow test gets a block that is the actual test.  The test_option could be called _argument instead -- it's about sending arguments to the test.

This was done this way to avoid processing the same test options over and over for the same card for each user, as was happening before (on wikirate, which had tons of users)